### ISSUE TITLE
bug: terrible issue with Celery tasks, worker wasn't doing anything!

### ISSUE THREAD

**Issue by mxmaxime** [bobby-home/bobby-home#284]
:warning: This is very problematic. I need to tackle this issue. Not acceptable at all.
### Specs
todo

### What is wrong?
```python
task.apply_async(blabla)
```
was called, not blocked (verified that with logs), but the worker wasn't doing anything. It was not receiving any tasks, so no processing neither.

- The concerned task (. security.camera_motion_detected) was found by the worker.
- `Connected to redis://redis-broker:6379/0`

```
celery_worker_1        | [2022-02-20 11:56:02,432: INFO/MainProcess] celery@81c10f6eb097 ready.                                                                                                
celery_worker_1        | [2022-02-20 11:56:02,447: INFO/MainProcess] Task hello_django.celery.periodic_check_pings[a3cd1b23-a76d-456c-a9d2-55d4b9155dca] received
celery_worker_1        | [2022-02-20 11:56:02,453: INFO/MainProcess] Task hello_django.celery.periodic_split_recordings[34f2d9f2-21c0-465e-bae2-143a4fbba63f] received
celery_worker_1        | [2022-02-20 11:56:02,459: INFO/ForkPoolWorker-2] split_recordings event_ref=b47124ed-a0f6-4f12-a22b-9516578b0a8a
celery_worker_1        | [2022-02-20 11:56:02,467: INFO/MainProcess] Task hello_django.celery.periodic_check_pings[8bc718c9-7fc7-4e57-a3b6-b152f586d3f2] received
celery_worker_1        | [2022-02-20 11:56:02,474: INFO/MainProcess] Task hello_django.celery.periodic_split_recordings[11862b8f-f15f-462b-a7ad-dfd8334b7c2d] received
celery_worker_1        | [2022-02-20 11:56:02,479: INFO/ForkPoolWorker-1] split_recordings event_ref=88a798bd-f797-4579-8102-4ae07a5940e8
celery_worker_1        | [2022-02-20 11:56:02,486: INFO/MainProcess] Task mqtt_services.tasks.verify_service_status[a500c25a-8652-4483-adf6-5feb30c099cd] received
celery_worker_1        | [2022-02-20 11:56:02,493: INFO/MainProcess] Task mqtt_services.tasks.verify_service_status[25ecaadc-97b5-4ac7-bb56-6fc72511829b] received
celery_worker_1        | [2022-02-20 11:56:02,498: INFO/MainProcess] Task hello_django.celery.periodic_check_pings[876cff77-95b0-42b7-8a51-aed22ae297fc] received
celery_worker_1        | [2022-02-20 11:56:02,503: INFO/MainProcess] Task security.camera_motion_detected[f48fb1f9-ec7a-4491-8be0-3c1c2b393f46] received
celery_worker_1        | [2022-02-20 11:56:02,509: INFO/MainProcess] Task security.camera_motion_picture[8c818bc8-e9df-477e-a2aa-fee6ee527dad] received
celery_worker_1        | [2022-02-20 11:56:02,517: INFO/MainProcess] Task mqtt_services.tasks.mqtt_status_does_not_match_database[cd3b8559-38e9-43f1-9a37-1276ab938bbb] received
celery_worker_1        | [2022-02-20 11:56:02,522: INFO/MainProcess] Task hello_django.celery.periodic_split_recordings[1f235b36-f0bb-446b-9c36-5e28ef5f160c] received
celery_worker_1        | [2022-02-20 11:56:02,527: INFO/MainProcess] Task hello_django.celery.periodic_check_pings[92a4c52c-e71d-442f-816c-991cfb706063] received
celery_worker_1        | [2022-02-20 11:56:02,532: INFO/MainProcess] Task hello_django.celery.periodic_split_recordings[7eaccb39-d635-4eac-bb96-55b2fbfd4480] received
celery_worker_1        | [2022-02-20 11:56:02,554: INFO/MainProcess] Task hello_django.celery.periodic_check_pings[eba3adb3-0124-4d1e-9181-3de192526b30] received
celery_worker_1        | [2022-02-20 11:56:02,575: INFO/MainProcess] Task hello_django.celery.periodic_split_recordings[4627e333-0b22-4c92-b1ba-1a48ee708cdc] received
celery_worker_1        | [2022-02-20 11:56:02,591: INFO/MainProcess] Task hello_django.celery.periodic_check_pings[ecb10d1f-db12-470f-9ce3-da5e6a4809ae] received
celery_worker_1        | [2022-02-20 11:56:02,593: INFO/ForkPoolWorker-4] Task hello_django.celery.periodic_check_pings[a3cd1b23-a76d-456c-a9d2-55d4b9155dca] succeeded in 0.13596566200021698s
: None                                         
celery_worker_1        | [2022-02-20 11:56:02,618: INFO/MainProcess] Task hello_django.celery.periodic_check_pings[935811e6-1264-4ce8-9830-4c9ae8275a49] received
celery_worker_1        | [2022-02-20 11:56:02,638: INFO/MainProcess] Task hello_django.celery.periodic_check_pings[24b25e74-1c46-4d9e-ae4b-1115d95e9c79] received

# later on
celery_worker_1        | [2022-02-20 11:56:02,662: INFO/MainProcess] Task mqtt_services.tasks.mqtt_status_does_not_match_database[242e72a4-d939-4d68-8520-5d02229e8cd5] received
celery_worker_1        | [2022-02-20 11:56:02,694: INFO/MainProcess] Task mqtt_services.tasks.mqtt_status_does_not_match_database[f6e7cdea-3909-4461-a17c-1a09fb0eb9c8] received
# ...
celery_worker_1        | [2022-02-20 11:56:02,725: INFO/MainProcess] Task mqtt_services.tasks.mqtt_status_does_not_match_database[01e6807f-8d78-4d66-88a6-6deb00f00dba] received
celery_worker_1        | [2022-02-20 11:56:02,762: INFO/ForkPoolWorker-3] Task hello_django.celery.periodic_check_pings[8bc718c9-7fc7-4e57-a3b6-b152f586d3f2] succeeded in 0.26014577900014046s
: None
# ...
celery_worker_1        | [2022-02-20 11:56:04,432: INFO/MainProcess] Task hello_django.celery.periodic_check_pings[40c3a7b4-32bc-4c8c-ae20-fa051a756028] received                              
celery_beat_1          | [2022-02-20 11:56:06,572: INFO/MainProcess] Scheduler: Sending due task hello_django.celery.periodic_check_pings() (hello_django.celery.periodic_check_pings)         
celery_worker_1        | [2022-02-20 11:56:07,107: INFO/MainProcess] Events of group {task} enabled by remote. 
```

and then, the issue:
```
python_process_mqtt_1  | on_motion_camera payload={'detections': [{'class_id': 'people', 'h': 251, 'score': 0.62109375, 'w': 185, 'x': 114, 'y': 48}], 'event_ref': '78f809ea-50c5-4fa8-a736-5a
a5e0da1016', 'status': True} topic=CameraMotionTopic(type='motion', service='camera', device_id='79dde14f')
python_process_mqtt_1  | creating background task                                              
python_process_mqtt_1  | created!
```
No logs about the worker! Between the logs "creating background task" and "created!" the `.apply_async` call was made.

### Expected behavior
Create a task and execute it.

### Actual behavior
Basically, nothing. No exception, no blocking call but the task was not created.
